### PR TITLE
fix: Update typescript playwright readme with failing test info

### PR DESCRIPTION
**Pull request by sfoslund**
#### Details

https://github.com/microsoft/axe-pipelines-samples/pull/850 added a failing test to this sample. This PR updates the readme to reflect this change. 

##### Motivation

Ensure documentation keeps up with code changes.

##### Context

Added separate screenshots for passing/ failing pipelines.

#### Pull request checklist

- [n/a] If this PR addresses an existing issue, it is linked: Fixes #0000
- [n/a] New sample content is commented at a similar verbosity as existing content
- [n/a] All updated/modified sample code builds and runs in at least one PR/CI build
- [n/a] PR checks for builds named `[failing example] ...` fail as expected

### CODE REVIEW

File: typescript-playwright-sample/README.md [hunk 1]
```diff
@@ -31,23 +31,44 @@ Some good places to start reading are:
...
-[![Build Status](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_apis/build/status/42?branchName=main)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/latest?definitionId=42&branchName=main)
-
 <!--
   Note to maintainers: The below example images/links come from a specific build instead of the most recent build so we can link to specific tabs.
   If you update the links such that they point to a different build, make sure to mark that build as Retained so the links don't expire in a month.
 -->
-The accessibility tests run as part of the `yarn test` build step:
 
-[![Screenshot of "yarn test" build logs in sample build](./assets/screenshot-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=22505)
+These two azure pipelines demonstrate what it looks like to incorporate passing or failing accessibility tests into your CI.
```
Comment (dbjorge) [inline typescript-playwright-sample/README.md:39]: nit:

```suggestion
These two Azure Pipelines demonstrate what it looks like to incorporate passing or failing accessibility tests into your CI.
```

File: typescript-playwright-sample/README.md [hunk 2]
```diff
@@ -31,23 +31,44 @@ Some good places to start reading are:
...
+
+[![Screenshot of "yarn test" build logs in passing sample build](./assets/screenshot-passing-logs-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169)
+
+The test pass results display in the Tests tab of the build logs:
+
+[![Screenshot of Tests tab in passing sample build](./assets/screenshot-passing-tests-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=ms.vss-test-web.build-test-results-tab)
+
+Detailed accessibility scan information also appears in the Scans tab, courtesy of the [SARIF SAST Scans Tab extension](https://marketplace.visualstudio.com/items?itemName=sariftools.scans):
+
+[![Screenshot of Scans tab in passing sample build](./assets/screenshot-passing-scans-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=sariftools.sarif-viewer-build-tab.sariftools.sarif-viewer-build-tab)
```
Comment (dbjorge) [inline typescript-playwright-sample/README.md:55]: It looks like the required `view` param changed when the sarif viewer extension got renamed, and we missed updating these links. I think this link should instead be:

```suggestion
[![Screenshot of Scans tab in passing sample build](./assets/screenshot-passing-scans-tab.png)](https://dev.azure.com/accessibility-insights/axe-pipelines-samples/_build/results?buildId=39169&view=sariftools.scans.build-tab)
```

(same for the failing sample build link on L71, and also in the typescript-selenium sample readme)

**Review by dbjorge — APPROVED** [review]
LGTM modulo the two minor comments. Thanks for retaining the builds you used already!

**Comment by sfoslund** [comment]
@dbjorge it looks like pushing new commits automatically dismisses reviews, mind re-approving this?

**Review by dbjorge — APPROVED** [review]

**Pull request merged**